tests/conftest.py

```diff
@@ -58,3 +58,142 @@ def app_config(app_config):
 def create_app():
     """Create app fixture for UI+API app."""
     return _create_app
+
+
+@pytest.fixture(scope='function')
+def full_record():
+    """
+    Dictionary to create a record with all the fields.
+
+    The following attributes are injected by the de/serialization:
+    - bucket
+    - recid
+    - conceptrecid
+    - files (when files are added to the record)
+    """
+    return {
+        "_visibility": True,
+        "_visibility_files": True,
+        "_owners": [1],
+        "_created_by": 2,
+        "_default_preview": "previewer one",
+        "_internal_notes": [{
+            "user": "inveniouser",
+            "note": "RDM record",
+            "timestamp": "2020-02-01"
+        }],
+        "embargo_date": "2022-12-31",
+        "contact": "info@inveniosoftware.org",
+        "community": {
+            "primary": "Maincom",
+            "secondary": ["Subcom One", "Subcom Two"]
+        },
+        "resource_type": {
+            "type": "image",
+            "subtype": "photo"
+        },
+        "identifiers": [{
+            "identifier": "10.5281/zenodo.9999999",
+            "scheme": "DOI"
+        }, {
+            "identifier": "9999.99999",
+            "scheme": "arXiv"
+        }],
+        "creators": [{
+            "name": "Julio Cesar",
+            "type": "Personal",
+            "given_name": "Julio",
+            "family_name": "Cesar",
+            "identifiers": [{
+                "identifier": "9999-9999-9999-9999",
+                "scheme": "Orcid"
+            }],
+            "affiliations": [{
+                "name": "Entity One",
+                "identifier": "entity-one",
+                "scheme": "entity-id-scheme"
+            }]
+        }],
+        "contributors": [{
+            "name": "Maximo Decimo Meridio",
+            "type": "Personal",
+            "given_name": "Maximo",
+            "family_name": "Decimo Meridio",
+            "identifiers": [{
+                "identifier": "9999-9999-9999-9998",
+                "scheme": "Orcid"
+            }],
+            "affiliations": [{
+                "name": "Entity One",
+                "identifier": "entity-one",
+                "scheme": "entity-id-scheme"
+            }],
+            "role": "RightsHolder"
+        }],
+        "titles": [{
+            "title": "A Romans story",
+            "type": "Other",
+            "lang": "eng"
+        }],
+        "descriptions": [{
+            "description": "A story on how Julio Cesar relates to Gladiator.",
+            "type": "Abstract",
+            "lang": "eng"
+        }],
+        "publication_date": "2020-06-01",
+        "licenses": [{
+            "license": "Copyright Maximo Decimo Meridio 2020. Long statement",
```

Comment:
Should this be the human readable title of the license (as opposed to copyright...)?

Reply:
Youp, just got in the mood for some Gladiator easter eggs.